openid-sharedsignals-framework-1_0.md

```diff
@@ -1232,7 +1232,7 @@ Errors are signaled with HTTP status codes as follows:
 
 | Code | Description |
 |------|-------------|
-| 400  | if the request cannot be parsed |
+| 400  | if the request cannot be parsed, delivery method not supported |
```

Comment:
This is a bit ambiguous. Did you mean "if the request cannot be parsed, or the delivery method is not supported"?